<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewTerrain</class>
 <widget class="QDialog" name="DialogNewTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Terreno</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Nombre</string>
       </property>
       <property name="buddy">
        <cstring>_le_name</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_le_name">
       <property name="text">
        <string>nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Terrenos</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="3">
         <widget class="QLabel" name="_l_tile_height">
          <property name="text">
           <string>00 px</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="_l_tile_widht">
          <property name="text">
           <string>00 px</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Alto de los tiles</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Ancho de los tiles</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="_pb_find">
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Examinar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLabel" name="_l_tools">
          <property name="maximumSize">
           <size>
            <width>331</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ancho del terreno</string>
       </property>
       <property name="buddy">
        <cstring>_sb_width</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_sb_width">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix">
        <string> tiles</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_l_wpx">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0 px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>86</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Alto del terreno</string>
       </property>
       <property name="buddy">
        <cstring>_sb_height</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_sb_height">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix">
        <string> tiles</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="_l_hpx">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>0 px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_pb_acept">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>76</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_pb_reject">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>76</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_le_name</tabstop>
  <tabstop>_pb_find</tabstop>
  <tabstop>_sb_width</tabstop>
  <tabstop>_sb_height</tabstop>
  <tabstop>_pb_acept</tabstop>
  <tabstop>_pb_reject</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_pb_find</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewTerrain</receiver>
   <slot>loadTools()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_pb_acept</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewTerrain</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_pb_reject</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewTerrain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadTools()</slot>
 </slots>
</ui>
